DigitalLearningSolutions.Data.Tests/DataServices/TutorialContentDataServiceTests/TutorialTests.cs

```diff
@@ -42,5 +42,62 @@ public void GetTutorialIdsByCourse_returns_correct_tutorials()
             // Then
             result.Should().BeEquivalentTo(expectedTutorials);
         }
+
+        [Test]
+        public void UpdateOrInsertCustomisationTutorialStatuses_updates_both_statuses_on_existing_CustomisationTutorial()
+        {
+            // When
+            using var transaction = new TransactionScope();
+            try
+            {
+                tutorialContentDataService.UpdateOrInsertCustomisationTutorialStatuses(49, CustomisationId, false, false);
+                var result = tutorialContentDataService.GetTutorialsBySectionId(SectionId, CustomisationId).ToList();
+
+                using (new AssertionScope())
+                {
+                    result.First().TutorialId.Should().Be(49);
+                    result.First().TutorialName.Should().Be("View documents");
+                    result.First().Status.Should().BeFalse();
+                    result.First().DiagStatus.Should().BeFalse();
+                }
+            }
+            finally
+            {
+                transaction.Dispose();
+            }
+        }
+
+        [Test]
+        public void UpdateOrInsertCustomisationTutorialStatuses_inserts_new_CustomisationTutorial()

```

Comment:
Could probably make this name more specific by appending "when_..." as the initial results are tested here and it makes it clear how the code should work.